test/test_distributions.py

```diff
@@ -3971,6 +3971,29 @@ def test_transformed_distribution_shapes(self):
             except NotImplementedError:
                 continue
 
+    def test_jit(self):
+        for transform in self.unique_transforms:
+            print('testing {}'.format(type(transform).__name__))
```

Comment:
Please don't print anything in the tests

Reply:
Ok I'll remove these, but it's really frustrating writing parametrized tests using `unittest`. What do you think of using PyTest #11578?